src/main/java/duke/DukeExceptions.java

```diff
@@ -0,0 +1,16 @@
+package duke;
+
+/**
+ * Placeholder Exception
+ */
+public class DukeExceptions extends Exception{

```

Comment:
DukeExceptions does not have any body, can be removed if you are not intending to work on it.